libcxxabi/src/demangle/ItaniumDemangle.h

```diff
@@ -890,6 +890,29 @@ class DynamicExceptionSpec : public Node {
   }
 };
 
+/// Represents the explicitly named object parameter.
+/// E.g.,
+/// \code{.cpp}
+///   struct Foo {
+///     void bar(this Foo && self);
+///   };
+/// \endcode
+class ExplicitObjectParameter final : public Node {
+  Node *Base;
+
+public:
+  ExplicitObjectParameter(Node *Base_)
+      : Node(KExplicitObjectParameter, Cache::Yes), Base(Base_) {
+    assert(Base != nullptr);
```

Comment:
The bots are complaining about `assert` here, looks like we're meant to use DEMANGLE_ASSERT now.

Reply:
Yea I'm a little confused about this one, since we're using plain `assert`s throughout the file. Any idea why specifically this one is failing?

Reply:
Maybe the bots are rebasing your changes on top of https://github.com/llvm/llvm-project/commit/c4779ea8e709a7bce7288988fabf1ba50e9c9477 ?

Reply:
ah good catch!